### ISSUE TITLE
[Godot] Add picture-in-picture camera previews to the editor when a Camera3D node is selected

### ISSUE THREAD

**Issue by MangoDragonHub** [Redot-Engine/redot-proposals#11]
### Tested versions

- Applies to all 4.x branches

### System information

All Avaliable Operating Systems

### Issue description

Since the Redot Proposal page isn't available yet I would like to use this a way to send my suggestion.
Since there is already a plug-in called Little Camera Preview in the add-ons library, it would be nice if this version of the engine natively had a camera preview when you select any camera in the scene. That way, it's easier for developers to iterate much faster without having to download a plugin, use the clunky Camera Preview button, or build the game.

### Steps to reproduce

Look at Little Camera preview's source code and re-implement it via GDExtention.

### Minimal reproduction project (MRP)

Currently no MRP available.

**Comment by TAGames** [comment]
I would go so far and say always show the camera preview when the current scene has a camera in it. Not only when selecting it. When there are multiple, maybe pick the first one?

**Comment by MangoDragonHub** [comment]
> I would go so far and say always show the camera preview when the current scene has a camera in it. Not only when selecting it. When there are multiple, maybe pick the first one?

Wouldn't that take away a bit screen real estate that way? Like if you go to a scene where there isn't a camera you have full view of the editor. Then if you go to a scene that has one or more cameras in the scene, there is always the tiny windows focuses on a random/high priority camera. At that point, we'll have a solution like Blender where you have to press a button or shortcut to switch active camera, which is not intuitive in my opinion for a Game Engine.

A good compromise is having the ability to collapse the view to get a bit of view back, but I still think having the camera preview show up when you select any camera and goes away when not.

**Comment by SkogiB** [comment]
Yeah, it shouldn't be an always-on feature, a toggle button for it would be fine.
